docs/guides/views.rst

```diff
@@ -358,6 +360,73 @@ Here we'll eliminate breadcrumbs that don't have at least one link.
 	    // returning nothing means "don't alter the returnvalue"
 	}
 
+View aliases
```

Comment:
I think these docs should be limited to explaining how existing views will be affected. They're a bit aspirational for a feature it's not clear we're going to add.